pkg/localstore/gc_test.go

```diff
@@ -534,3 +535,194 @@ func TestSetTestHookCollectGarbage(t *testing.T) {
 		t.Errorf("got hook value %v, want %v", got, original)
 	}
 }
+
+func TestPinAfterMultiGC(t *testing.T) {
+	db := newTestDB(t, &Options{
+		Capacity: 100,
+	})
+
+	pinnedChunks := make([]swarm.Address, 0)
+
+	// upload random chunks above db capacity to see if chunks are still pinned
+	for i := 0; i < 2000; i++ {
```

Comment:
if the capacity is 100, why do you need to insert 6000?

Reply:
That's the test case,...  The issue occurs when we bombard chunks greater that `dbCapacity` ..more than once...  2000 is a big number for CI, May be we can reduce it to 200.